app/src/main/java/io/zenandroid/onlinego/mygames/MyGamesPresenter.kt

```diff
@@ -182,8 +182,11 @@ class MyGamesPresenter(
     private fun onNotification(notification: JSONObject) {
         if(notification["type"] == "gameOfferRejected") {
             notificationsRepository.acknowledgeNotification(notification)
-            val message = if(notification.has("message")) "Message is:\n\n${notification["message"]}" else ""
-            view.showMessage("Bot rejected challenge", "This might happen because the opponent's maintainer has set some conditions on the challenge parameters. $message")
+            val message = if(notification.has("message") && notification["message"].toString() != "null") "Message is:\n\n${notification["message"]}" else ""
```

Comment:
Relying on the name of the game is a bit of a hack, although I cannot think of a proper way of doing it better...

Reply:
yes :) felt dirty doing it, but the notification content doesn't have much else to play with.. should be OK as I suspect we'll be in full control of the game name for the time being